guide/how-it-works/node.md

```diff
@@ -136,18 +152,83 @@ Connecting to a single node controlled by a third-party is a convenient setup fo
    layout = "float-left"
 %}
 
-Third-party APIs should be avoided. These are custom connection mechanisms built on top of block data, and there is no way to validate the authenticity of the data.
+Third-party APIs should be avoided. These are custom connection mechanisms built on top of nodes, and there is no way to validate the authenticity of the data.
+
+</div>
+
+## Lightning nodes
+
+Depending on the use case, lightning nodes can be operated in different ways. Some need consistent uptime, large amounts of [liquidity]({{ '/guide/how-it-works/liquidity/' | relative_url }}), and to be well connected with other lightning nodes. Others are only periodically online or interacting with the network.
+
+A lightning nodes primary purpose is to track payment channel states, and calculate routes for payments to take through lightning. A lightning node needs to be online to receive and send payments.
+
+Currently, Self-custody on lightning requires a user to run their own node and regularly come online to monitor their payment channels for cheating attempts. Work is being done to separate custody and node operation with projects like [Greenlight](https://blockstream.com/lightning/greenlight/) and [Validating Lightning Signer (VLS)](https://gitlab.com/lightning-signer).
+
+### Routing nodes
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/how-it-works/node/routing-node.png"
+   retina = "/assets/images/guide/how-it-works/node/routing-node@2x.png"
+   alt-text = ""
+   width = 100
```

Comment:
Images are 101x101. Forcing 100x100 makes them look blurry on non-retina screens. I suggest re-exporting the images at 100x100.

Reply:
Looks fine on my retina screen. But if it looks blurry on yours, best to play it safe. That 1 pixel value isn't easily divisible.

@Bosch-0 if you're re-exporting them, might be good to get rid of the strange hairline border. That might be contributing to that extra pixel.

<img width="293" alt="Screen Shot 2022-07-07 at 7 33 24 AM" src="https://user-images.githubusercontent.com/2414177/177763967-80978014-2a67-477b-a4f1-226bd6f2fed1.png">